symbolic/am_or.m

```diff
@@ -1,6 +1,11 @@
 function fun = am_or(a,b)
-% syms x y
-% f = symfun(sym('cw_or(x,y)'),[x y]);
-% fun = f(a,b);
+% am_or is the amici implementation of the symbolic or function
+%
+% Parameters:
+%  a: first input parameter @type sym
+%  b: second input parameter @type sym
+%
+% Return values:
+%  fun: logical value, negative for false, positive for true
 fun = am_max(a,b);
```

Comment:
with the mapping

true -> positive
false -> negative

maximum is only positive (true) if at least one of a and b is positive (true)